frontend/packages/data-portal/app/components/Download/ConfigureDownloadContent.tsx

```diff
@@ -9,12 +9,16 @@ import { ConfigureTomogramDownloadContent } from './ConfigureTomogramDownloadCon
 export function ConfigureDownloadContent() {
   const { t } = useI18n()
   const { datasetTitle, runName, objectName } = useDownloadModalContext()
-  const { annotationId, objectShapeType } = useDownloadModalQueryParamState()
+  const { annotationName, annotationId, objectShapeType } =
```

Comment:
could we just derive the name from DownloadModalContextValue.annotationToDownload.id and DownloadModalContextValue.annotationToDownload.annotation.object_name?

also, bit confused cuz I didn't see `setAnnotationName()` called anywhere 🤔

Reply:
oh it's calling `openAnnotationDownloadModal()` 👍

Reply:
don't we already have `annotationToDownload` though? (recently renamed from `activeAnnotation` 😬)